packages/graph-editor/src/components/contextMenus/selectionContextMenu.tsx

```diff
@@ -28,51 +23,6 @@ export const SelectionContextMenu = ({ id, nodes }: INodeContextMenuProps) => {
   );
   const selectedNodeIds = selectedNodes.map((node) => node.id);
 
-  const onGroup = useCallback(() => {
```

Comment:
We should not be removing the group, we should be storing it in the serialized graph using a finalizer

Reply:
@mck can we get on the same page on this one? From my understanding this was a temporary decision, right?

Reply:
I don't think saving groups is a short task. Currently we are not adding the group node to the graph, we only add it to `reactFlowInstance`